torch/testing/_internal/common_nn.py

```diff
@@ -1647,6 +1647,20 @@ def fractional_max_pool3d_test(test_case):
         desc='2d_affine_large_feature',
         test_cpu=False,
     ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(3, 48, 0.0),
+        cpp_constructor_args='torch::nn::GroupNormOptions(3, 48).eps(0.0)',
+        input_fn=lambda: torch.rand(2, 48, 15, 15),
```

Comment:
Do you think this size is enough to check accuracy?

Reply:
A smoke test case for large input added.